android/build.gradle

```diff
@@ -122,6 +122,6 @@ def kotlin_version = getExtOrDefault('kotlinVersion')
 dependencies {
     //noinspection GradleDynamicVersion
     api 'com.facebook.react:react-native:+'
-    implementation 'com.revenuecat.purchases:purchases-hybrid-common:1.6.3'
+    implementation 'com.revenuecat.purchases:purchases-hybrid-common:1.7.0'
```

Comment:
eeek just noticed this one isn't up to 1.7.1. i can include with canmakepayments?

Reply:
ugh its the second time this happens to us... Ill do a patch since canMakePayments is a minor right?

Reply:
good point, yeah it is as minor. maybe we could add pre-commit hook to check that ios and android are the same? i can look into this in my automation ticket

Reply:
I am actually going to write a quick script to increase all these versions so we dont miss any file